receiver/zipkinreceiver/config.go

```diff
@@ -4,22 +4,89 @@
 package zipkinreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zipkinreceiver"
 
 import (
+	"fmt"
+
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/config/confighttp"
+	"go.opentelemetry.io/collector/confmap"
+	"go.opentelemetry.io/collector/featuregate"
+)
+
+var disallowHTTPDefaultProtocol = featuregate.GlobalRegistry().MustRegister(
+	"zipkinreceiver.httpDefaultProtocol.disallow",
+	featuregate.StageAlpha,
+	featuregate.WithRegisterDescription("When enabled, usage of the default http configuration is disallowed"),
```

Comment:
Can we refer to the original config as "top-level"? Does it make sense to you?
```suggestion
	featuregate.WithRegisterDescription("When enabled, usage of the top-level HTTP configuration is disallowed"),
```